src/shared-components/dialogModal/style.ts

```diff
@@ -31,7 +31,9 @@ export const Overlay = styled.div`
   }
 `;
 
-export const ModalContainer = styled.div`
+export const ModalContainer = styled.div<{
+  backgroundColor: string;
```

Comment:
[Nit] Since this is always coming from a parent there's no requirement for specificity but I think it makes sense to be consistent and make this `Colors['background']` as well.